app/uk/gov/ons/sbr/models/enterprise/EnterpriseLink.scala

```diff
@@ -0,0 +1,13 @@
+package uk.gov.ons.sbr.models.enterprise
+
+import io.swagger.annotations.ApiModelProperty
+import play.api.libs.json.Json
+
+case class EnterpriseLink(
+  @ApiModelProperty(value = "Enterprise Reference Number (ERN)", dataType = "string", example = "1000000012", required = true) ern: Ern,
```

Comment:
Swagger ApiModelProperty can be moved to Ern cc that way other classes can use it with the swagger annotation

Reply:
Having tried this, it unfortunately fails to identify that Ern / Lurn are simple value types that map should map directly to a JsString rather than a JsObject.  The "example" JSON generated is then incorrect.